controller/model/authenticator_mod_cert.go

```diff
@@ -81,13 +89,55 @@ func (module *AuthModuleCert) Process(context AuthContext) (string, error) {
 				}
 			}
 
-			return authenticator.IdentityId, nil
+			opts := x509.VerifyOptions{
+				Roots:         module.getRootPool(),
+				Intermediates: x509.NewCertPool(),
+				KeyUsages:     []x509.ExtKeyUsage{x509.ExtKeyUsageAny},
+			}
+
+			if _, err := cert.Verify(opts); err == nil {
+				return authenticator.IdentityId, nil
+			} else {
+				println(err)
```

Comment:
shouldn't this be a log msg and not a println?

Reply:
Leftover from debugging, removed.